Sucuri/sucuri-wordpress-plugin
# Fixed remote fonts usage

This PR fixes the issue of google fonts being loaded/used by the plugin. The update replaces the google fonts with local/self-hosted fonts.

### How to test
- Pull the branch
- `$ npm install` 
- Start the `wp-env` instance`$ npx npx wp-env start`
- Load `http://localhost:8888/wp-admin/` and login with the `wp-env` admin user (user: `admin`, password: `password`)
- Navigate to `http://localhost:8888/wp-admin/admin.php?page=sucuriscan#auditlogs` 
- All fonts should be loading and rendering correctly.
